modules/UI/videolayout/SmallVideo.js

```diff
@@ -542,35 +539,42 @@ SmallVideo.prototype.showRaisedHandIndicator = function (show) {
         return;
     }
 
-    var indicatorSpanId = "raisehandindicator";
-    var indicatorSpan = this.getIndicatorSpan(indicatorSpanId);
-
-    indicatorSpan.innerHTML
-        = "<i id='indicatoricon' class='icon-raised-hand'></i>";
-
-    // adds a tooltip
-    UIUtil.setTooltip(indicatorSpan, "raisedHand", "top");
-    APP.translation.translateElement($(indicatorSpan));
+    var indicatorSpan = this.getIndicatorSpan({
+        id: 'raisehandindicator',
+        content: '<i id="indicatoricon" class="icon-raised-hand"></i>',
+        tooltip: 'raisedHand'
+    });
 
-    $(indicatorSpan).css("visibility", show ? "visible" : "hidden");
+    $(indicatorSpan)[show ? "show" : "hide"]();
 };
 
 /**
- * Gets (creating if necessary) the "indicator" span for this SmallVideo
-  identified by an ID.
+ * Gets (creating if necessary) the "indicator" span for this SmallVideo.
+ * 
+ * @param options.id {String} element ID
+ * @param options.content {String} HTML content of the indicator
+ * @param options.tooltip {String} The key that should be passed to tooltip
+ *
+ * @returns {HTMLElement} the raised hand indicator
  */
-SmallVideo.prototype.getIndicatorSpan = function(id) {
-    var indicatorSpan;
-    var spans = $(`#${this.videoSpanId}>[id=${id}`);
-    if (spans.length <= 0) {
-        indicatorSpan = document.createElement('span');
-        indicatorSpan.id = id;
-        indicatorSpan.className = "indicator";
-        $('#' + this.videoSpanId)[0].appendChild(indicatorSpan);
-    } else {
-        indicatorSpan = spans[0];
+SmallVideo.prototype.getIndicatorSpan = function(options) {
+    var indicator = this.container.querySelector('#' + options.id);
+
+    if (indicator) {
+        return indicator;
     }
-    return indicatorSpan;
+
+    indicator = document.createElement('span');
+    indicator.className = 'indicator';
+    indicator.id = options.id;
+
+    indicator.innerHTML = options.content;
+
+    UIUtil.setTooltip(indicator, options.tooltip, "top");
```

Comment:
I think it's good to add a comment that this element will be translated by UIUtil.setTooltip and that's why it is not translated here. Also you can add a description in the comment why we are doing this.